package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -83,12 +83,22 @@ def _load_data_helper(
         A tuple containing the data catalog, pipeline dictionary and dataset stats dictionary.
     """
 
-    with KedroSession.create(
-        project_path=project_path,
-        env=env,
-        save_on_close=False,
-        extra_params=extra_params,
-    ) as session:
+    try:
+        kedro_session = KedroSession.create(
+            project_path=project_path,
+            env=env,
+            save_on_close=False,
+            runtime_params=extra_params,  # type: ignore[call-arg]
+        )
+    except TypeError:
+        kedro_session = KedroSession.create(
+            project_path=project_path,
+            env=env,
+            save_on_close=False,
+            extra_params=extra_params,
+        )
+
+    with kedro_session as session:
```

Comment:
I think you could get around this duplication, by just not using the named arguments.